### ISSUE TITLE
Value of aria-current attribute in breadcrumbs should be "page" instead of "Page"

### ISSUE THREAD

**Issue by dpodolkhov** [semrush/intergalactic#912]
**Describe the bug**
Active breadcrumb sets aria-current="Page", which is not a valid value for this attribute.
The value should be "page" in lower case and of course should not be localized.

[A clear and concise description of what the bug is.](https://github.com/semrush/intergalactic/blob/master/semcore/breadcrumbs/src/Breadcrumbs.jsx#L58)

**Code example**

```html
<a class="___SBreadcrumbsItem_tji0e_gg_ __active_tji0e_gg_" data-ui-name="Breadcrumbs.Item" tabindex="-1" aria-current="页码" disabled="">this example</a>
```

**To Reproduce**
[Inspect a sandbox](https://codesandbox.io/s/thirsty-haze-8xvl5t)

**Expected behavior**
aria-current for active breadcrumb should be always "page"

**Comment by ilyabrower** [comment]
Thank you, we've fixed it.
